velox/core/QueryConfig.h

```diff
@@ -18,6 +18,24 @@
 #include "velox/core/Config.h"
 
 namespace facebook::velox::core {
+enum class CapacityUnit {
```

Comment:
Can you add testing for this? Thanks!

Reply:
These are moved from native code to velox as it is a common config util that is needed here and there both places. But I can add tests for this

Reply:
I think we shall add test for these utilities? Thanks!